handlers/easy_info/__init__.py

```diff
@@ -0,0 +1,12 @@
+from eventhandler import EventHandler
+
+
+class EasyInfoHandler(EventHandler):
+    def on_issue_labeled(self, api, payload):
+        new_label = payload['label']['name']
+
+        if new_label == 'E-Easy':
```

Comment:
This won't work if the label has a different capitalization. Should it be solved using `payload['label']['name'].lower() == 'e-easy'` or make the label to look for a per-repository configuration?